### PR TITLE
Add basic coverage configuration

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

**Comment by bwrsandman** [comment]
There seems to be a bug with `coverage --include` and pypy3

**Comment by pedrobaeza** [comment]
What can we do then?

**Comment by bwrsandman** [comment]
We can wait for pypy3 or the coverage people to fix this bug. This isn't critical.

**Comment by pedrobaeza** [comment]
But we will get a red cross in all PRs until then...

**Comment by bwrsandman** [comment]
We wont. The only red is in this PR where we try to add --include.

The downside is that the coverage won't be right.

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# .coveragerc to control coverage.py
+[run]
+branch = True
+include =
+    setup.py
+    openupgradelib/*
```
Comment (bwrsandman) [inline .coveragerc]: Failing in pypy3 causing it to stall in `FnmatchMatcher.match`

**Comment by pedrobaeza** [comment]
OK then :+1:

**Comment by bwrsandman** [comment]
Related issues:
* https://bitbucket.org/ned/coveragepy/issues/347/coverage-pypy3-2to3-is-unacceptably-slow
* https://bitbucket.org/pypy/pypy/issues/1871/10x-slower-than-cpython-under-coverage

**Comment by pedrobaeza** [comment]
Thanks for the info

**Comment by bwrsandman** [comment]
I think I have worked around the issue.

**Comment by bwrsandman** [comment]
\o/

Again ignore the decrease in coverage

**Comment by pedrobaeza** [comment]
Cool, I see!

**Pull request merged**